docs/plans/plans-crowdloan-datahighway-kusama-tanganika.md

```diff
@@ -0,0 +1,232 @@
+* Submitted our Collator Wasm and Genesis and were Onboarded as a Parathread and then as a Parachain on Rococo in Polkadot.js Apps 
+
+## What is Tanganika Network's DHX Token
+
+DHX token is the key to participating in the IoT-related DAO, Inter-Chain Data Market, the LPWAN Roaming Hub, and associated cross-chain DApps.
+
+DHX token is the native token of Tanganika Network.
+
+### Token Utility
+
+* DAO Governance - To participating in DHX Mining you must be an active stakeholder and participate in community governance. For details, see the [10:1 Rule](https://dev.datahighway.com/docs/whitepapers/shortpaper#the-101-rule) in the DataHighway Shortpaper.
+* Collator Rewards
+* Cross-Chain DApps
+* Cross-Chain Payments
+* Inter-Chain Data Market
+* LPWAN Roaming Hub
+
+### Distribution
+
+The following is the proportion of the total supply of DHX that will be distributed as rewards:
+
+* 0.3% - Crowdloan Contributors
+  * This represents 1% of the 30,000,000 DHX in the DHX DAO Unlocked Reserves (Treasury endowment for DHX Mining Rewards), which comprise 30% of the DataHighway's total supply of 100,000,000 DHX 
+
+DHX Mining Power (mPower) Rewards equivalent to a 40% mPower boost (based on 24 month period) shall apply to tokens that are contributed to the crowdloan.
+
+Note: In the crowdloan, those that lock their tokens to help the Tanganika Network secure a parachain slot will need to lock them for up to 48 weeks in a Kusama parachain crowdloan in order to get DHX tokens in return.
+
+Depending on the vesting period mentioned in the Reward Distributions section, they could bond the DHX tokens they claim from DHX Mining when the tokens are distributed to them.
+
+The tokens they lock during the crowdloan period will still earn them an mPower boost to increase their DHX Mining rewards.
+
+### Mining
+
+See the [DataHighway Shortpaper](https://dev.datahighway.com/docs/whitepapers/shortpaper).
+
+Note: The minimum amount of mPower and bonded DHX that is required to mine DHX tokens through DHX Mining increases every 3 months.
+
+Secure DHX Mining may be achieved using M2 Pro hardware.
+
+The different forms of DHX Mining that will be available are mentioned in the whitepaper (i.e. hardware mining using an M2 Pro, token mining by locking DataHighway's authorized value added tokens like MXC, E-IOTA, Polkadot, Filecoin, Huobi BTC).
+
+### How to get DHX tokens
+
+* Collating (running a Parachain Collator)
+* DHX Mining (with funding from DAO Unlocked Reserves)
+  * DataHighway's authorized value added tokens supported include: MXC, E-IOTA, Polkadot, Filecoin, Huobi BTC
+* Crowdloan Contribution
+* Market Exchange
+
+### Technology
+
+Tanganika Network as a IoT layer for Web 3.0 metaverse developers.
```

Comment:
make it as hyperlink which can open Tanganika description or some info....

Reply:
a hyper to what?
what's the link to the specific "Tanganika description" that you're referring to?
what's the link to other "info" that you want a hyperlink to point to?

Reply:
> what's the link to the specific "Tanganika description" that you're referring to?

yes link to documents ? who knows what is this Tanganika, we need to point out the write link. as this name is given by us.

Reply:
> what's the link to other "info" that you want a hyperlink to point to?

same link

Reply:
the name was suggested by Piotr. i'll update the document to say "The Tanganika Network was named after the second largest of the lakes in eastern Africa."

i still don't know what specific Tanganika documents you mean. please let me know the link if any documents exist